extensions/todo-list/changelog.md

```diff
@@ -1,5 +1,9 @@
 # Todo List Changelog
 
+## Fix Handling URLs Without a Scheme (2022-03-17)
+
+Previously, URLs without a scheme would crash the extension -- these are now handled by adding the `http` scheme if there is no scheme.
+
 ## Opening URLs and Editing Todos (2022-03-10)
```

Comment:
```suggestion
## [Opening URLs and Editing Todos] - 2022-03-10
```

Reply:
We added strict format of the [changelog](https://developers.raycast.com/basics/prepare-an-extension-for-store#version-history) - could you address this?